silk-rules/src/main/scala/org/silkframework/rule/execution/TransformReportBuilder.scala

```diff
@@ -27,7 +27,7 @@ private class TransformReportBuilder(rules: Seq[TransformRule], previousReport:
 
     val updatedRuleResult =
       if(currentRuleResult.sampleErrors.size < maxSampleErrors) {
-        val values = rule.sourcePaths.map(entity.evaluate)
+        val values = rule.sourcePaths.map(p => entity.evaluate(Path(p.operators)))  //TODO TypedPath change:  why do source paths provide TypedPaths? PAY SPECIAL ATTENTION TO THIS SECTION!
```

Comment:
Good question, why sourcePaths returns typed paths. This should be reconsidered.